health-services/project-factory/src/server/api/campaignApis.ts

```
@@ -734,6 +773,61 @@
 
+async function getEmployeesBasedOnUserName(dataToCreate: any[], request: any) {
+  const searchBody = {
+    RequestInfo: request?.body?.RequestInfo,
+  };
+
+  // const tenantId = request?.body?.ResourceDetails?.tenantId;
+  const searchUrl = config.host.hrmsHost + config.paths.hrmsEmployeeSearch;
+  logger.info(`Waiting for 10 seconds`);
+  // await new Promise((resolve) => setTimeout(resolve, 10000));
+  const chunkSize = 50;
+  let foundUsernames = new Set<string>(); // ✅ Initialize resultSet properly
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'foundUsernames' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)